library/Fission/AWS.hs

```diff
@@ -41,16 +40,3 @@ validate changeSet =
 
   where
     status = changeSet ^. crrsrsResponseStatus
-
--- | Ensure that a request completed, and that the status code is not in an error range
-ensureContent
```

Comment:
The resp already gets validated in `Route53.changeRecord`

And now `registerDomain` returns `m (Either ServerError AWS.DomainName)`. We can just `ensureM` in the route function.

If I'm missing something here please let me know! But with the refactor this function seems redundant